tests/edgeless.spec.ts

```diff
@@ -1317,6 +1319,47 @@ test('resize element which attaches connector', async ({ page }) => {
   await assertEdgelessHoverRect(page, [160, 200, 140, 100]);
 });
 
+test('resize connector by capital resize handler', async ({ page }) => {
```

Comment:
We'd better split `edgeless.spec.ts` into multiple `/edgeless/*.spec.ts`.

Reply:
Sure, I will split it by features.